internal/cli/get.go

```diff
@@ -120,10 +124,23 @@ func (o *GetOptions) Validate(args []string) error {
 		if kind != DeviceKind && kind != FleetKind {
 			return fmt.Errorf("owner can only be specified when fetching devices and fleets")
 		}
-		if (kind == DeviceKind || kind == FleetKind) && len(name) > 0 {
+		if len(name) > 0 {
 			return fmt.Errorf("cannot specify owner together with a device or fleet name")
 		}
 	}
+	if o.Summary || o.SummaryOnly {
```

Comment:
@avishayt @fzdarsky, please share your opinion about the current integration to the CLI.
Keep in mind that "get fleets/{name}" also has DevicesSummary inside the FleetStatus field, which only displays the device count in the chosen fleet.